src/content/docs/guides/platform-setup/configure-internal-services.mdx

```diff
@@ -58,7 +60,9 @@ TENZIR_PLATFORM_INTERNAL_TENANT_TOKEN_ENCRYPTION_KEY=
 TENZIR_PLATFORM_INTERNAL_APP_API_KEY=
 ```
 
-## Internal Proxy
+## Tenzir UI
+
```

Comment:
It's generally bad practice to start two headings subsequently without a single sentence in between.